io.c

```diff
@@ -8557,6 +8557,12 @@ rb_io_printf(int argc, const VALUE *argv, VALUE out)
     return Qnil;
 }
 
+VALUE
+argf_printf(int argc, const VALUE *argv, VALUE out)
+{
+  return rb_io_printf(argc, argv, out)
```

Comment:
Missing semicolon at the end:

```suggestion
  return rb_io_printf(argc, argv, out);
```